webhook_to_fedora_messaging_messages/github/github.py

```diff
@@ -0,0 +1,96 @@
+# SPDX-FileCopyrightText: 2024 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+from ..base import Webhook2FedMsgBase
+from utils import summarize_repository_event
```

Comment:
If you're importing from a local module, it should be `from .utils import`, with the dot. Otherwise Python will try to import from the standard library first, and it may conflict.